lumicks/pylake/tests/test_file.py

```diff
@@ -257,3 +257,61 @@ def test_invalid_access(h5_file):
 
         with pytest.raises(IndexError):
             m["Kymo1"]
+
+
+def test_h5_export(tmpdir_factory, h5_file):
```

Comment:
The testing of the private and public API is kinda mixed up. If you want to test both, consider parameterizing on that. For example:

```py
def _private_api(file, *args, **kwargs):
    return write_h5(file.h5, *args, **kwargs)

def _public_api(file, *args, **kwargs):
    return file.export_h5(*args, **kwargs)

@pytest.mark.parametrize("func", [_internal_api, _public_api])
def test_h5_export(tmpdir_factory, h5_file, func):
```

Reply:
Fair point. Changed! 👍